### DISCUSSION TITLE
Which tools support graphql-config?

### DISCUSSION THREAD

**Discussion by borekb** [kamilkisiela/graphql-config#590] [General] [answered]
The docs say:

> Think about GraphQL Config as **one configuration for all your GraphQL tools**.

Which tools are they?

**Comment by kamilkisiela** [comment, marked as answer]
GraphQL Codegen, GraphQL Inspector, GraphQL CLI, Amplify-CLI, VSCode extensions, Webstorm GraphQL Extension, Talk, eslint plugins, GraphiQL and few small libraries and projects.
